.github/workflows/release.yaml

```diff
@@ -62,7 +62,6 @@ jobs:
         # Non-deprecated from https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
         # Note: We don't test arm64 on release as it is unlikely to fail and too much effort.
         include:
-          - os: ubuntu-18.04
```

Comment:
maybe switch this to 22.04 and add a trailing comment to the next line like `  # envoy x.x is the earliest to support this`.

Also mention in the PR desc which versions no longer are testable even if it doesn't matter as we only test ones released?